package.json

```diff
@@ -21,10 +21,12 @@
     "lint:fix": "eslint 'src/**/*.ts' --fix",
     "clean": "rm -rf dist dts",
     "test": "npm run lint",
-    "build": "tsc",
+    "build": "npm run prisma-generate && tsc",
```

Comment:
The Prisma client should be generated after `yarn install`, so I don't think this is necessary.